tests/components/mqtt/test_discovery.py

```diff
@@ -1545,6 +1545,7 @@ class TestFlow(config_entries.ConfigFlow):
 
         async def async_step_mqtt(self, discovery_info: MqttServiceInfo) -> FlowResult:
             """Test mqtt step."""
+            await asyncio.sleep(0.1)
```

Comment:
Ideally we freeze time so the test doesn't take longer

Reply:
I'll have a look

Reply:
Did some test with freezer enabled, but without success, so I'll keep the sleep. in place for now.